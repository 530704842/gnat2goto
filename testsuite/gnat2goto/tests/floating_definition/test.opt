ALL XFAIL gnat2goto fails with type definition and type declaration failure
