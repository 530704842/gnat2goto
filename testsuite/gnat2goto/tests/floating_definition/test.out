[assertion.1] Range Check: SUCCESS
[1] file floating_definition.adb line 10 : SUCCESS
[2] file floating_definition.adb line 11 : FAILURE
VERIFICATION FAILED
